erc/ERCS/erc-7562.md

```diff
@@ -342,3 +437,7 @@ effectively censoring the other sender.
            and the only mitigation at the moment is to bump the gas price of the censored `UserOperation`.
         2. There should be at least one "proper" bundler in the network, that follows rule [STO-041],
```

Comment:
1. As it is now, STO-041 is written backwards and actually enables this attack.
2. Let's discuss this section. I'm not sure it's valuable the way it's written right now.